src/main/java/me/moodcat/api/models/RoomModel.java

```diff
@@ -0,0 +1,8 @@
+package me.moodcat.api.models;
+
+/**
+ * @author Jan-Willem Gmelig Meyling
+ */
+public class RoomModel {
+
+}
```

Comment:
Currently the `RoomAPI` still abuses the `Room` entity as model. This should be refactored, but before this we should know which attributes are actually required for the `RoomModel` (see: https://github.com/MoodCat/MoodCat.me-Core/blob/chat-backend/src/main/java/me/moodcat/api/RoomAPI.java#L106)